dist/webstomp.js

```diff
@@ -74,7 +74,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	    // This method creates a WebSocket client that is connected to
 	    // the STOMP server located at the url.
 	    client: function client(url) {
-	        var options = arguments.length <= 1 || arguments[1] === undefined ? { protocols: _utils.VERSIONS.supportedProtocols() } : arguments[1];
+	        var options = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : { protocols: _utils.VERSIONS.supportedProtocols() };
```

Comment:
it appears that the build is generating new logic for conditional tests. I am adding below comments on the lines I actually changed